src/main/java/de/leonhard/storage/Toml.java

```diff
@@ -61,15 +62,15 @@ public Toml(final File file) {
   // ----------------------------------------------------------------------------------------------------
 
   @Override
-  protected final Map<String, Object> readToMap() throws IOException {
+  protected final @NotNull Map<String, Object> readToMap() throws IOException {
     return TomlManager.read(getFile());
   }
 
   @Override
-  protected final void write(final FileData data) {
+  protected final void write(final @NotNull FileData data) {
     try {
       TomlManager.write(data.toMap(), getFile());
-    } catch (final IOException ioException) {
+    } catch (final @NotNull IOException ioException) {
```

Comment:
When an exception is thrown it will not be null, this is unneeded.

Reply:
Nullity refactor mistake.